packages/aws-cdk/lib/cli/telemetry/sink/file-endpoint-sink.ts

```diff
@@ -0,0 +1,28 @@
+import type { EndpointTelemetrySinkProps } from './endpoint-sink';
+import { EndpointTelemetrySink } from './endpoint-sink';
+import type { FileTelemetrySinkProps } from './file-sink';
+import { FileTelemetrySink } from './file-sink';
+import type { TelemetrySchema } from '../schema';
+
+export interface FileEndpointTelemetrySinkProps extends FileTelemetrySinkProps, EndpointTelemetrySinkProps {
+}
+
+export class FileEndpointTelemetrySink {
```

Comment:
Just make this a MultiSinkSink (Waterworks? Pumphouse? TelemteryDistributionNetwork?) that takes in any number of pre-configured sinks and emits and flushes to all of them.